RemoteSignTool/RemoteSignTool.Client/Program.cs

```diff
@@ -204,7 +206,13 @@ static int Main(string[] args)
 
         private static async Task<string> CommunicateWithServer(string archivePath, string signSubcommands)
         {
-            using (var client = new HttpClient())
+            var progressHandler = new ProgressMessageHandler();
```

Comment:
I'm not sure if it's completely necessary and correct but we can try to put ProgressMessageHandler in the using.